evaluator/opa.go

```diff
@@ -75,23 +75,26 @@ func NewOpaEvaluator(ctx context.Context, log *logp.Logger, cfg config.Config) (
 	// provide the OPA configuration which specifies
 	// fetching policy bundles from the mock bundleServer
 	// and logging decisions locally to the console
-	config := []byte(fmt.Sprintf(opaConfig, ServerAddress, cfg.Evaluator.DecisionLogs))
+	opaCfg := []byte(fmt.Sprintf(opaConfig, ServerAddress, cfg.Evaluator.DecisionLogs))
 
 	// create an instance of the OPA object
 	opaLogger := newEvaluatorLogger()
 	opaDecisionLogger := newDecisionLogger()
 	opa, err := sdk.New(ctx, sdk.Options{
-		Config:        bytes.NewReader(config),
+		Config:        bytes.NewReader(opaCfg),
 		Logger:        opaLogger,
 		ConsoleLogger: opaDecisionLogger,
 	})
+
 	if err != nil {
 		return nil, fmt.Errorf("fail to init opa: %s", err.Error())
 	}
 
-	rules, err := cfg.GetActivatedRules()
-	if err != nil {
-		log.Warnf("failed to get activated rules: %v", err)
+	var rules *config.Benchmarks
+	if cfg.RuntimeCfg != nil {
```

Comment:
I am wondering about this code again, if the config was able to propagate here with no `RuntimeCfg` we are in a serious problem.

We should have one validation layer (as we do have now), and I guess that no need for extra validation here.
On the other hand, it does not hurt to have extra validation here.

Up to u

Reply:
No `RuntimeCfg` will result in running all rules, so no issue imo.